docs/en/building/basic-usage.md

```diff
@@ -0,0 +1,118 @@
+# Basic Usage
+
+## Creating a Dojo application
+
+The first step for building a Dojo application is to install the CLI
+
+> Command line
```

Comment:
We can probably skip these quotes for CLI commands; they're only really useful when naming source code files.